packages/mds-types/device.ts

```diff
@@ -0,0 +1,39 @@
+import { Enum, Timestamp, UUID } from './utils'
+import { VEHICLE_STATE } from './vehicle/vehicle_states'
+import { VEHICLE_TYPE } from './vehicle/vehicle_types'
+
+export const PROPULSION_TYPES = Enum('human', 'electric', 'electric_assist', 'hybrid', 'combustion')
+export type PROPULSION_TYPE = keyof typeof PROPULSION_TYPES
+export const RIGHT_OF_WAY_STATES = ['available', 'reserved', 'non_operational', 'trip'] as const
```

Comment:
No corresponding type for `RIGHT_OF_WAY_STATES`?

Reply:
Confirmed with Neil that this can be deleted